.stylelintrc.json

```diff
@@ -4,7 +4,7 @@
 		"at-rule-no-unknown": [
```

Comment:
Maybe convert this to a .js file as well to have the same format for config files for ESLint, Stylelint and Prettier.